alert/views.py

```diff
@@ -1,15 +1,16 @@
-import re
 import base64
+import json
 import logging
 
+from django.conf import settings
+from django.http import Http404, HttpResponse, HttpResponseRedirect, JsonResponse
 from django.shortcuts import render
-from django.http import HttpResponseRedirect, JsonResponse, Http404, HttpResponse
 from django.urls import reverse
 from django.views.decorators.csrf import csrf_exempt
 
-from .models import *
-from .tasks import generate_course_json, send_course_alerts
-from options.models import get_bool
+from alert.models import Registration, RegStatus, record_update, register_for_course, update_course_from_record
```

Comment:
I'll be sad that lint doesn't like relative imports or `*` imports but it's probably for the better.